File: TP3/Exo4.py
# Définition de la fonction
def maximum_de_trois(a, b, c):
    return max(a, b, c)

# Programme principal
x = int(input("Entrez le premier nombre : "))
y = int(input("Entrez le deuxième nombre : "))
z = int(input("Entrez le troisième nombre : "))

max_val = maximum_de_trois(x, y, z)
print("Le maximum est :", max_val)

File: TP3/Exo4.js
// Définition de la fonction
function maximumDeTrois(a, b, c) {
    return Math.max(a, b, c);
}

// Programme principal
let x = parseInt(prompt("Entrez le premier nombre :"));
let y = parseInt(prompt("Entrez le deuxième nombre :"));
let z = parseInt(prompt("Entrez le troisième nombre :"));

let maxVal = maximumDeTrois(x, y, z);
console.log("Le maximum est : " + maxVal);
